From b288e8c48496c9027629e89f4b0607b047646cb1 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <michael.i.doherty@intel.com>
Date: Wed, 20 Jan 2016 12:54:14 +0000
Subject: [PATCH] Use correct gettext version

Signed-off-by: Ikey Doherty <michael.i.doherty@intel.com>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 09b81e6..a8e625e 100644
--- a/configure.ac
+++ b/configure.ac
@@ -180,7 +180,7 @@ dnl
 dnl Gettext
 dnl
 AM_GNU_GETTEXT([external],[need-ngettext])
-AM_GNU_GETTEXT_VERSION([0.18.1])
+AM_GNU_GETTEXT_VERSION([0.19])
 
 AC_PROG_RANLIB
 
-- 
2.7.0

